Src/Couchbase.Linq/Extensions/BucketExtensions.cs

```diff
@@ -8,7 +9,8 @@ public static class BucketExtensions
     {
         public static IQueryable<T> Queryable<T>(this IBucket bucket)
         {
-            return EntityFilterManager.ApplyFilters(new BucketQueryable<T>(bucket));
+            //TODO refactor so ClientConfiguration is injectable
+            return EntityFilterManager.ApplyFilters(new BucketQueryable<T>(bucket, new ClientConfiguration()));
```

Comment:
@jeffrymorris,

I feel like this point here is where we should pull ClusterHelper configuration as the default, instead of a new configuration.  Otherwise we've lost support for pulling the ContractResolver from the current cluster configuration when using ClusterHelper.GetBucket("default").Queryable<T>().  It will always be the default CamelCasePropertyNamesContractResolver contract resolver.

Ideally, I think it would be great to have some way to get the contract resolver from IBucket.  Maybe have CouchbaseBucket implement an additional IBucketConfiguration interface that we could query for the configuration, something like that.  But that requires changes to the core client, which is obviously more difficult.

Brant